server/internal/config_parser.go

```diff
@@ -10,14 +10,15 @@ import (
 
 // Configuration struct to hold app configurations
 type Configuration struct {
-	Server     Server      `json:"server"`
-	MailSender MailSender  `json:"mailSender"`
-	Database   DB          `json:"database"`
-	Token      JwtToken    `json:"token"`
-	Account    GridAccount `json:"account"`
-	Version    string      `json:"version"`
-	Salt       string      `json:"salt"`
-	Admins     []string    `json:"admins"`
+	Server      Server      `json:"server"`
+	MailSender  MailSender  `json:"mailSender"`
+	Database    DB          `json:"database"`
+	Token       JwtToken    `json:"token"`
+	Account     GridAccount `json:"account"`
+	Version     string      `json:"version"`
+	Salt        string      `json:"salt"`
+	Admins      []string    `json:"admins"`
+	NotifyHours int         `json:"notifyHours"`
```

Comment:
I like the pattern in general `$description$interval`, but I'd change this field to `notifyAdminsIntervalHours` to remove the ambiguity